src/style.css

```diff
@@ -0,0 +1,108 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  height: 403px;
+  background-color: #fff;
+  font-weight: normal;
+  font-size: 11px;
+  line-height: 12.89px;
+  padding: 30px 14px 15px;
+}
+
+.card__image {
+  align-self: center;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  height: 31px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 15.47px;
+  padding-top: 45px;
+  margin: 0 auto;
+}
+
+.card__code {
+  padding-top: 36px;
+}
+
+.card__rate {
+  display: flex;
+  padding-top: 15px;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.card__stars {
+  display: flex;
+}
+
+.card__star {
+  background-image: url(/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+}
+
+.stars--1 .card__star:nth-child(-n+1),
+.stars--2 .card__star:nth-child(-n+2),
+.stars--3 .card__star:nth-child(-n+3),
+.stars--4 .card__star:nth-child(-n+4),
+.stars--5 .card__star:nth-child(-n+5) {
+  background: url("images/star-active.svg") center no-repeat;
+}
+
+.card__prices {
+  display: flex;
+  padding: 21px 0;
+  justify-content: space-between;
+  align-items: baseline;
+}
+
+.card__coast {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18.75px;
+}
+
+.card__button {
+  border: 1px solid #00acdc;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 40px;
+  background-color: #00acdc;
+  text-decoration: none;
+  line-height: 16px;
+  color: #fff;
+  border-radius: 5px;
+  text-transform: uppercase;
+  font-weight: 700;
+  font-size: 14px;
+}
+
+.card__button:hover {
+  color: #00acdc;
+  background-color: #fff;
+  border: 1px solid #00acdc;
```

Comment:
If border is the same with and without hover, then why do you set this rule again in card__button:hover?